yogstation/code/game/objects/items/implants/implant_mindslave.dm

```diff
@@ -0,0 +1,97 @@
+/obj/item/implant/mindslave
+	name = "mindslave implant"
+	desc = "Now YOU too can have your very own mindslave! Pop this implant into anybody and they'll obey any command you give for around 15 to 20 minutes."
+	activated = 0
+	var/permanent = FALSE
+	var/time = 10200 // 1020 seconds aka 17 mins
+	var/timerid
+	var/datum/objective/protect/protect_objective
+
+/obj/item/implant/mindslave/get_data()
+	var/dat = {"<b>Implant Specifications:</b><BR>
+				<b>Name:</b> Syndicate Mindslave implant MK1<BR>
+				<b>Life:</b> Varies between 15 and 20 minutes.<BR>
+				<b>Important Notes:</b> Personnel injected with this device become loyal to the user and will obey any command given for a limited time.<BR>
+				<HR>
+				<b>Implant Details:</b><BR>
+				<b>Function:</b> Allows user to command anyone implanted to do whatever they want.<BR>
+				<b>Special Features:</b> Person with implant MUST obey any order you give. <BR>
+				<b>Integrity:</b> Implant will last around 15 and 20 minutes."}
+	return dat
+
+/obj/item/implant/mindslave/implant(mob/target,mob/user,silent=0)
+	if(target.mind)
+		if(target == user)
```

Comment:
make it not work on changelings, like prebase https://github.com/yogstation13/yogstation/pull/1070